src/utils/event-manager/event-manager.js

```diff
@@ -16,6 +16,8 @@ function ManagerMock(m) {
   instance.off = chainedNoop;
   instance.destroy = chainedNoop;
   instance.emit = chainedNoop;
+  instance.get = chainedNoop;
```

Comment:
This mock is incorrect -- `get()` should return a value, not the instance.
See also [deck.gl](https://github.com/uber/deck.gl/blob/master/src/controls/events/event-manager.js#L16)...is it time to break off `event-manager` into a module/package shared between these libs? /cc @ibgreen @Pessimistress